pom.xml

```
@@ -4,6 +4,7 @@
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
         <version>4.86</version>
+        <relativePath />
```

@jonesbusy BTW the extra space is only necessary for plugins still using `maven-release-plugin`, since it has a buggy XML round-trip parser/generator that would forcibly reinsert it on every release. For plugins using CD,
```suggestion
        <relativePath/>
```
suffices. (XML never needed a space here; sometimes XHTML documents used it, to work around very old buggy web browsers which did not actually include XML parsers and tried to parse as HTML.)